test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampProviderTests.cs

```diff
@@ -24,7 +24,11 @@ namespace NuGet.Packaging.FuncTest
     [Collection(SigningTestCollection.Name)]
     public class TimestampProviderTests
     {
+#if IS_DESKTOP
         private const string ArgumentNullExceptionMessage = "Value cannot be null.\r\nParameter name: {0}";
+#else
+        private const string ArgumentNullExceptionMessage = "Value cannot be null. (Parameter '{0}')";
+#endif
```

Comment:
I'd rather that you simplify this than make it more complicated.  See my recommendation below.

Reply:
Thanks! Fixed.